examples/nlp/md/multiple_choice_task_with_transfer_learning.md

```diff
@@ -572,91 +619,147 @@ def build_model():
 model = build_model()
 ```
 
+<div class="k-default-codeblock">
+```
+Downloading from https://www.kaggle.com/api/v1/models/keras/deberta_v3/keras/deberta_v3_extra_small_en/2/download/config.json...
+
+```
+</div>
+    
+  0%|                                                                                                                                                         | 0.00/539 [00:00<?, ?B/s]
+
+    
+100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 539/539 [00:00<00:00, 1.16MB/s]
+
+    
+
+
+<div class="k-default-codeblock">
+```
+Downloading from https://www.kaggle.com/api/v1/models/keras/deberta_v3/keras/deberta_v3_extra_small_en/2/download/model.weights.h5...
+
+```
+</div>
+    
+  0%|                                                                                                                                                        | 0.00/270M [00:00<?, ?B/s]
+
+    
+100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 270M/270M [00:08<00:00, 33.4MB/s]
+
+
 Let's checkout the model summary to have a better insight on the model.
 
 
 ```python
 model.summary()
 ```
 
+<div class="k-default-codeblock">
+```
+Model: "model"
```

Comment:
This summary is the old style summary. The example was converted to Keras 3 already, and now it seems to run on Keras 2.

Reply:
Surprising that it is using Keras 2, the issue reported here has some problem with the plot model rendering https://github.com/keras-team/keras-io/issues/1713, and the original PR created in keras_core does not seem to have migrated to Keras 3 https://github.com/keras-team/keras-core/pull/884

Reply:
Make sure to `pip install keras --upgrade` in your environment before generating the files.